geoAssembler/widgets/nb_tabs.py

```diff
@@ -195,13 +194,13 @@ def _move_quadrants(self, pos):
         d = pos['owner'].description.lower()[0]
         pn = int(pos['new'])
         po = int(pos['old'])
-        sign = np.sign(pn - po)
+        delta = pn - po
```

Comment:
wouldn't it be more save to keep the signum here?

Reply:
I think that if you type a number in a text box, you can make the x or y pos change by more than 1px at a time. If you take the sign, then a change will only ever move it 1px, so the display position will get out of sync with the text boxes.